addons/website_crm_partner_assign/models/res_partner.py

```diff
@@ -60,7 +60,7 @@ def _compute_partner_weight(self):
     def _compute_opportunity_count(self):
         super()._compute_opportunity_count()
         opportunity_data = self.env['crm.lead'].with_context(active_test=False)._read_group(
-            [('partner_assigned_id', 'in', self.ids)],
+            [('partner_assigned_id', 'in', self.ids), ('partner_id', 'not in', self.ids)],
```

Comment:
Actually we should probably follow the computation done in base module: also bubble up results to the parent, isn't it ? In which case I think we could simply extract the domain into a method that we would override here. That way we do a single read_group and computation, just the domain is extended here. What do you think ?

Reply:
Hello @tde-banana-odoo 

We can try that :smiley: 

Thanks!!